packages/api/test/stats-get.spec.js

```diff
@@ -0,0 +1,35 @@
+import assert from 'assert'
+import { createClientWithUser, DBTestClient } from './scripts/helpers.js'
+import { fixtures } from './scripts/fixtures.js'
+
+describe('Get Stats', () => {
+  /** @type{DBTestClient} */
+  let client
+
+  before(async () => {
+    client = await createClientWithUser()
+  })
+
+  it('should return proper response for cid v1', async () => {
+    const demoData = {
+      deals_size_total: 169389985753391,
```

Comment:
These are taken directly from hard coded seed data inside of the cargo.testing.sql file